src/main/java/uk/gov/hmcts/reform/unspec/launchdarkly/OnBoardingOrganisationControlService.java

```diff
@@ -0,0 +1,39 @@
+package uk.gov.hmcts.reform.unspec.launchdarkly;
+
+import lombok.RequiredArgsConstructor;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.stereotype.Service;
+import uk.gov.hmcts.reform.prd.model.Organisation;
+import uk.gov.hmcts.reform.unspec.service.OrganisationService;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Optional;
+
+@Slf4j
+@Service
+@RequiredArgsConstructor
+public class OnBoardingOrganisationControlService {
+
+    public static final String ORG_NOT_ONBOARDED = "You cannot use this service because your organisation is"
+        + " not part of the HMCTS civil damages pilot.";
+
+    private final FeatureToggleService featureToggleService;
+    private final OrganisationService organisationService;
+
+    public List<String> validateOrganisation(String userBearer) {
+        Optional<Organisation> userOrganisation = organisationService.findOrganisation(userBearer);
+
+        Boolean organisationOnboarded = userOrganisation
```

Comment:
do we need this as boxed `Boolean` ? Prefer `boolean` here to avoid NPE.

Reply:
done